### PR TITLE
[semver:skip] Updated orb-tools version 

### PR DESCRIPTION
**Pull request by Jaryt**
### Checklist

- [ ] All new jobs, commands, executors, parameters have descriptions
- [ ] Examples have been added for any significant new features
- [ ] README has been updated, if necessary

### Motivation, issues

Orb tools version contained deprecated macos executor

### Description 

Orb tools updated to 10.0.5

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -15,6 +15,20 @@ parameters:
...
+executors:
+  macos:
+    macos:
+      xcode: 12.5.1
+  base:
+    docker:
+      - image: cimg/base:stable
+  openjdk:
+    docker:
+      - image: cimg/openjdk:16.0.2
```
Comment (gmemstr) [inline .circleci/config.yml:27]: Why are we specifically testing this against an openjdk image?

Comment (gmemstr) [reply .circleci/config.yml:27]: If it's a case of "because we were before", let's just get rid of it.

**Review by gmemstr — CHANGES_REQUESTED** [review]

**Comment by Jaryt** [comment]
I agree with the jdk image. Really no reason to test on that when the base of the image is already being tested on. Replaced that with alpine, should be good to go.

**Review by KyleTryon — APPROVED** [review]
Agreed. We might want to update that alpine image later but im good with this.

**Review by gmemstr — APPROVED** [review]

**Pull request merged**